### DISCUSSION TITLE
Mouse won't work

### DISCUSSION THREAD

**Discussion by Cyanwolf2014** [rustdesk/rustdesk#8027] [General]
### Bug Description

I was streaming my friends computer and he was gone, so I went to turn on his game and I can't push any buttons on the game itself, so I sent to files and I can press anything. Just move the blue dot around. What do I do?

### How to Reproduce

Nothing I have to say

### Expected Behavior

Nothing I have to say

### Operating system(s) on local side and remote side

Win 10

### RustDesk Version(s) on local side and remote side

Newest

### Screenshots

Nothing I have to say

### Additional Context

Nothing I have to say

**Comment by xlionjuan** [comment]
Some games will deny the access of the remote desktop software

**Reply by Cyanwolf2014** [reply]
But it used to work and then stopped

**Reply by xlionjuan** [reply]
Could you try another remote software to check it is still blocked or not? And ensure your RustDesk is installed, because some games are using kernel-level anti-cheat and game itself will running at administrator, software that are not running at administrator can't control the software that runs with administrator.

And also provide the name of the game.

**Reply by 78957898** [reply]
> 你可以尝试使用其他远程软件来检查它是否仍然被阻止访问。确保你的RustDesk已经安装，因为有些游戏使用了内核级别的反作弊技术，游戏本身会以管理员身份运行，没有以管理员身份运行的软件无法控制以管理员身份运行的软件。
> 
> 并且请提供游戏的名字。

天堂W
